tempto-core/src/main/java/io/prestosql/tempto/internal/hadoop/hdfs/WebHdfsClient.java

```diff
@@ -212,6 +212,13 @@ public String getOwner(String path)
         return ((String) getAttributeValue(path, "owner"));
     }
 
+    @Override
+    @SuppressWarnings("unchecked")
+    public String getPermission(String path)
+    {
+        return ((String) getAttributeValue(path, "permission"));
```

Comment:
can you add a test for (maybe into `tempto-examples`)?

Reply:
nit: unnecessary parenthesis

Reply:
Let me add the test.